app/styles/ui/history/_expandable-commit-summary.scss

```diff
@@ -189,4 +195,15 @@
       }
     }
   }
+
+  .selectable {
+    user-select: text;
+    cursor: text;
+
+    * {
+      user-select: unset;
+      pointer-events: unset;
+      cursor: text;
+    }
```

Comment:
A little duplicate of a css rule above to make the emails use a "cursor" for easily selecting/copying. I may remove the duplicate code above when complete the remainder of this work in favor of this helper class. But, for now separating concerns. :)